index.html

```diff
@@ -37,13 +37,19 @@ <h1 class="animated fadeInUp leftAlign">SUGAR: A COLLABORATIVE,<br> OPEN-SOURCE
         <div class="row">
             <div class="col-md-12" >
                 <div class="row">
-                    <div class="col-md-5 col-md-push-1">
+                    <div class="col-md-10 col-md-push-1">
                         <div class="block">
                             <div class="section-title">
                                 <h2>SUGAR</h2>
                                 <p>Learning software for children</p>
                             </div>
-                            <p class="customColor">Sugar is an open-source learning platform that reinvents how computers are used for education. Collaboration, reflection, and discovery are integrated directly into the user interface. Sugar promotes "studio thinking" and "reflective practice". Through Sugar's clarity of design, children and teachers have the opportunity to use computers on their own terms. Students can reshape, reinvent, and reapply both software and content into powerful learning activities. Sugar's focus on sharing, criticism, and exploration is grounded in the culture of free software (FLOSS).</p>
+                            <p class="customColor">Sugar is an activity-focused, open-source learning platform for children. 
+                                Collaboration, reflection, and discovery are integrated directly into the user interface. 
+                                Through Sugar's clarity of design, children and teachers have the opportunity to use 
+                                computers on their own terms. Students can reshape, reinvent, and reapply both 
+                                software and content into powerful learning activities. 
+                                Sugar's focus on sharing, criticism, and exploration is grounded in the 
+                                culture of free software (FLOSS).</p>
```

Comment:
Maybe move the end tag off the line content?

Happy to see once sentence per line like god intended.